Simplified/AxisService/NYPLContentDownloader.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  NYPLContentDownloader.swift
+//  Simplified
+//
+//  Created by Raman Singh on 2021-04-22.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+protocol NYPLAxisContentDownloading {
+  func downloadItem(from url: URL,
+                    _ completion: @escaping (Result<Data, Error>) -> Void)
+}
+
+
+/// A content downloader which stops downloading content upon receiving a failure.
+class NYPLContentDownloader: NYPLAxisContentDownloading {
```

Comment:
I have a question about this class, is this an Axis specific class or are we going to expand this for other DRMs?
`NYPLContentDownloader` sounds like a general class name but the `init` and `networkExecutor` below are tied to an Axis related protocol.

Reply:
Great point. Changed to NYPLAxisContentDownloader